### PR TITLE
pythonPackages.pytest-localserver: 0.3.5 -> 0.3.7

### LINKED ISSUE #26237: pytest-localserver broke unstable
## Issue description

`pytest-localserver` broke unstable for me.
Full backtrace below.

### Steps to reproduce

Rebuild on a908ad6fd3, which is unstable atm.

## Technical details

* System: 17.09pre107928.65477dcd34 (Hummingbird)
* Nix version: nix-env (Nix) 1.11.9
* Nixpkgs version: "17.09pre108224.a908ad6fd3"
* Sandboxing enabled: build-use-sandbox = true

Pinging maintainer of pytest-localserver @DamienCassou 

```
builder for ‘/nix/store/xkwr7cprpzjxkqnm5vmpbyybww9fz0aq-python3.6-pytest-localserver-0.3.5.drv’ failed with exit code 1
cannot build derivation ‘/nix/store/w7h84s8dhq9vfb0zaid7hvy4rjjipx93-vdirsyncer-0.15.0.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/gki59xi3k0vn75z3rhzz00q3plx698qi-khal-0.9.5.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/jj9gipn3xb40chxw4c4z1yms975ixn3b-system-path.drv’: 2 dependencies couldn't be built
cannot build derivation ‘/nix/store/rf8aiq5x6z9kkxxkg6h4c9p33d5ya9qk-dbus-1.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/8x9ydlzn5fayrcj4cv26mqlh70bz8zph-unit-polkit.service.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/8014zzjpxhz127q8lyj30196liixq5fa-unit-systemd-fsck-.service.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/n3z4jlq2xaghq23nb54plg2lf3b6ml5j-xsession.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/9jmqm81jd01sa4x1d000vgs8g6dq9k0r-desktops.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/kkrf5mr3g5ilc5nsi205s2n7x1agbm1j-unit-dbus.service.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/vszfd80crn5zka3j8w7xfna2xxsaip24-sddm.conf.drv’: 2 dependencies couldn't be built
cannot build derivation ‘/nix/store/zbnd72rvnbwanzrgpziq1smjfgnc94xi-system-units.drv’: 3 dependencies couldn't be built
cannot build derivation ‘/nix/store/7qqb0va2l0ajblh3jhzinbsy24ar7nzz-user-units.drv’: 1 dependencies couldn't be built
cannot build derivation ‘/nix/store/505m9bb09rxn542sxk9nq1yxjrgp931a-etc.drv’: 4 dependencies couldn't be built
cannot build derivation ‘/nix/store/ka4wc87wj8k7fldlapiqdny0q0ymshxp-nixos-system-terry-17.09pre108224.a908ad6fd3.drv’: 2 dependencies couldn't be built
error: build of ‘/nix/store/ka4wc87wj8k7fldlapiqdny0q0ymshxp-nixos-system-terry-17.09pre108224.a908ad6fd3.drv’ failed
```

### PR DESCRIPTION
**Pull request by matthiasbeyer**
## Warning

This patch is based on the `nixos-unstable` branch.

###### Motivation for this change

Closes #26237 

###### Things done

- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

Artificially tested (by building). Going to rebuild my system with this patch now and reporting back then.

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -5281,11 +5281,11 @@ in {
 
   pytest-localserver = buildPythonPackage rec {
     name = "pytest-localserver-${version}";
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: Can you move this expression to `pkgs/development/python-modules/pytest-localserver/default.nix`. See other files there on what to do with `pname`.

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -5281,11 +5281,11 @@ in {
...
-    version = "0.3.5";
+    version = "0.3.7";
 
     src = pkgs.fetchurl {
```
Comment (FRidh) [inline pkgs/top-level/python-packages.nix]: fetchPypi

**Review by FRidh — CHANGES_REQUESTED** [review]
No need to rebuild after this change, its a maintenance release.

**Comment by matthiasbeyer** [comment]
I'd be happy to do these changes, but I'd prefer to open another PR for them. This is simply a pkg update PR to fix the build on unstable.

**Pull request merged**